TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/subsystems/WebcamAprilTags.java

```diff
@@ -0,0 +1,215 @@
+package org.firstinspires.ftc.teamcode.robot.subsystems;
+
+import android.annotation.SuppressLint;
+import android.util.Size;
+
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
+import org.firstinspires.ftc.teamcode.core.lib.autonomousControl.Pose2d;
+import org.firstinspires.ftc.teamcode.core.lib.gamepad.GamepadManager;
+import org.firstinspires.ftc.teamcode.core.lib.interfaces.Subsystem;
+import org.firstinspires.ftc.teamcode.robot.constants.AutonomousConstants;
+import org.firstinspires.ftc.vision.VisionPortal;
+import org.firstinspires.ftc.vision.apriltag.AprilTagDetection;
+import org.firstinspires.ftc.vision.apriltag.AprilTagLibrary;
+import org.firstinspires.ftc.vision.apriltag.AprilTagLibrary.Builder;
+import org.firstinspires.ftc.vision.apriltag.AprilTagProcessor;
+import org.openftc.easyopencv.OpenCvInternalCamera;
+
+import java.util.List;
+
+public class WebcamAprilTags implements Subsystem {
+    private static WebcamAprilTags instance;
+    public enum Sections{
```

Comment:
add enum to it's own file

Reply:
I had no idea enums could be separate files, I shall do that promptly